src/components/LotteryList/LotteryList.jsx

```diff
@@ -0,0 +1,13 @@
+import './LotteryList.css';
+import {lotteryList} from '../Config/LotteryList';
+const LotteryList = () => {
+    return (
+        <div className='LotteryList'> 
+          <h4> ထီဆုကြီးများ</h4>
+           {lotteryList.map( lottery => 
+              <p> {lottery}</p>
+           )}
```

Comment:
- add key in lotteryList mapping
{lotteryList.map((lottery, key) => 
              <p key={key}> {lottery}</p>
 )}